tests/test_resource_manager.rs

```diff
@@ -93,7 +93,7 @@ impl TestResourceManager {
 
         // k8s seems to need a bit of time for namespace permissions to flow
         // through the system.  TODO: make this less worse
-        tokio::time::delay_for(tokio::time::Duration::from_millis(100)).await;
+        tokio::time::delay_for(tokio::time::Duration::from_millis(1000)).await;
```

Comment:
Note: This is just a temporary fix. Given the number of namespaces we are creating, it often takes the token controller more than 100ms to create all the default tokens. I will create an issue to fix this with something more robust

Reply:
https://github.com/deislabs/krustlet/issues/403